doc/userguide/rules/flow-keywords.rst

```diff
@@ -340,6 +340,28 @@ Signature example::
 
  alert ip any any -> any any (msg:"Flow has 20 packets"; flow.pkts_toclient:20; sid:1;)
 
+flow.pkts_either
+------------------
+
+Flow number of packets in either direction (integer)
```

Comment:
I wonder if it is needed to add that it is either, but not the some of both - or if it is clear enough.

Reply:
I think mathematically `either` is different than `both` or/and `sum`. But, if it seems like it can confuse new people, sure. Do you have suggestions on incorporating it in the sentence?